spec/openapi.yaml

```diff
@@ -1433,11 +1441,85 @@ paths:
+        
+          Alpha tags are made up of 3-11 letters and/or numbers. Alpha tags must be registered and approved before sending, and must have clear relevancy to your business and/or use case.
+        
+          Alpha Tags appear as the From number when you receive messages.
+        
+        ### What is a good Alpha Tag? 
+        
+        Valid use cases include:
+        * Business names
+        * Trademark names    
+        * Product or service name
+        * an acronym, initialism, or contraction of your entity
+        
+        ### In addition to the new requirements around clearly relating to the business, we typically advise the following for alpha tags to ensure maximum compatibility with the various carriers:
+        * 6-11 characters long
+        * Only contains characters from the following sets:
+        * A-Z
+        * a-z
+        * 0-9
+        * _ (underscore)
+        * \- (hyphen)
+        
+        ## Sender address is my own number
+        Before you can send messages using a your own number (i.e. a non-dedicated or rotary number), you need to verify that you have a right to use that number. Ensuring you have a right to use a phone number is an important ACMA regulatory requirement, aiming to prevent scam, spam and misuse of messaging services.
+        
+        If you do not wish to verify a number from within the web portal, or need to verify a large quantity of numbers, you can verify a number programmatically - via API.
+        
+        ### The process to verify a number via the API is:
+        1. (If you don’t have existing credentials) - Create Basic or HMAC credentials for the API by going to https://hub.messagemedia.com/api-settings
+        2. Submit a verification request with the number and destination country (usually AU) as per below. **Note**: *the verification is for the account associated with the API credentials only.*
+        3. A unique verification code will be sent to the number
+        4. Submit an approval request containing the verification code form that number. A 200 OK response will indicate the number has been verified and is ready for use.
+
+        ⚠️ Own numbers need to be re-verified every 12 months. You will be notified by email that verification of your number is about to expire.
+        
+        ## The most basic request has the following structure:
           - ```sender_address``` is case insensitive. When one already exists for this account, you get a error message ```conflict```;
           - ```destination_countries``` currently we only support ```AU``` and ```GB``` as the destination country;
-          - ```sender_address_type``` and ```usage_type``` must be ```ALPHANUMERIC```;
+          - ```sender_address_type``` and ```usage_type``` must be ```ALPHANUMERIC```and ```ALPHANUMERIC``` for alpha tag, ```INTERNATIONAL```and ```OWN_NUMBER``` for my own number;
```

Comment:
I don't really understand this sentence. Can we make it a bit clearer?

Reply:
Thanks, fixed